app/shared/taginput/taginput.component.html

```diff
@@ -0,0 +1,13 @@
+﻿<div>
+    <tag-input [ngModel]="[]"
```

Comment:
Putting static empty array as input as ngModel lead to always empty fields, even if there is data to show.

Better is to put a variable which contains the model, in this case a String Array, which contains the user email addresses. 
Also consider, that ngModel has Two-Way Binding, so it should be something like` [(ngModel)]="userEmailList"`.